test/00.hello_world/test_serialize_driver.py

```diff
@@ -0,0 +1,47 @@
+import freetensor as ft
+import pytest
+from random import randint
+
+
+def test_basic():
+    l, r = -2147483648, 2147483647
+    for i in range(10000):
```

Comment:
No need to test against 10000 numbers. Running time of tests also matters.

In a unit test, you test against some arbitrary number, for example 3. If your program does well against 3, normally it won't fail against 7, unless these two numbers are really different :)